schemaregistry/schemaregistry.yaml

```diff
@@ -0,0 +1,313 @@
+        '201':
+          description: Created
+          headers:
+            Location:
+              schema:
+                type: string
+        '409':
+          description: Schema group already exists
+      tags:
+      -  'groups'
+    delete:
+      summary: Delete schema group
+      description: Delete schema group in schema registry namespace.
+      operationId: deleteGroup
+      responses:
+        '204':
+          description: OK no content
+        '404':
+          description: Specified group not found
+      tags:
+      -  'groups'
+  /schemagroups/{group-name}/schemas:
+    parameters:
+    - name: group-name
+      in: path
+      description: schema group
+      required: true
+      schema:
+        type: string
+    get:
+      tags:
+      - 'groups'
+      summary: Get schemas for group name
+      description: Returns schema by group name.
+      operationId: getSchemasByGroup
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json;format=avro:
+              schema:
+                type: array
+                items: 
+                  type: string
+        '404':
+          description: Group not found
+    delete:
+      tags:
+      - 'groups'
+      summary: Deletes all schemas in group
+      description: Deletes all schemas under specified group name.
+      operationId: deleteSchemasByGroup
+      responses:
+        '204':
+          description: OK no content
+        '404':
+          description: Group not found
+  /schemagroups/{group-name}/schemas/{schema-name}:
+    parameters:
+      - name: group-name
+        in: path
+        description: schema group
+        required: true
+        schema:
+          type: string
+      - name: schema-name
+        in: path
+        description: schema name
+        required: true
+        schema:
+          type: string
+    post: 
+      summary: Register schema
+      description: >
+        Register schema.
+        If schema of specified name does not exist in specified group, schema is created at version 1.
+        If schema of specified name exists already in specified group, schema is created at latest version + 1.
+        If schema with identical content already exists, existing schema's ID is returned. 
+      operationId: createSchema
+      requestBody:
+        description: schema content
+        required: true
+        content:
+          application/json;format=avro:
+            schema:
+              $ref: '#/components/schemas/SchemaBytePayload'
+      responses:
+        '200':
+          $ref: '#/components/responses/SchemaIdResponse'
+        '400': 
+          description: Invalid request
+      tags:
+        - 'runtime'
+    get:
+      summary: Get latest version of schema
+      description: Get latest version of schema. 
+      operationId: getLatestSchema
+      responses: 
+        '200':
+          $ref: '#/components/responses/SchemaBytePayloadResponse'
```

Comment:
Do you want to include the ID that was served in this response, so that clients can retrieve the value again on subsequent calls?